requirements.txt

```diff
@@ -45,8 +44,7 @@ geographiclib==1.50       # via geopy
 geopy==1.22.0             # via apache-superset (setup.py)
 gunicorn==20.0.4          # via apache-superset (setup.py)
 humanize==2.4.0           # via apache-superset (setup.py)
-idna-ssl==1.1.0           # via aiohttp
-idna==2.9                 # via email-validator, idna-ssl, yarl
+idna==2.9                 # via email-validator
```

Comment:
There seem to be changes to this file that shouldn't be here. make sure you're rebuilding this from a rebased master.